apps/src/lib/kits/maker/ui/SetupInstructions.jsx

```diff
@@ -97,6 +114,79 @@ class ConnectionInstructions extends React.Component {
   }
 }
 
+class CPExpressInstallInstructions extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    // TODO: layout with images
+    // TODO: get screenshot for image copy
+    // TODO: reset button gif- put in our cdn?
+    // TODO: make i18n strings for all the strings
+    return (
+      <div id="CPExpressInstallInstructions">
+        <h2>Install the Firmata firmware onto the CP Express</h2>
+        <ol>
+          <li>
+            Set your Circuit Playground Express to Bootloader Mode
+            <div style={style.oneColumn}>
+              <ul>
+                <li>
+                  Press or double-press the RESET button in the center of the
+                  board.
+                </li>
+                <li>
+                  You've successfully entered bootloader mode when the LEDs turn
+                  green and your computer detects a new removable storage device
+                  called 'CPLAYBOOT'.
```

Comment:
When you convert to i18n strings, I would leave 'CPLAYBOOT' out of the localized string.

Reply:
[Here's](https://github.com/code-dot-org/code-dot-org/blob/d185420ee51c89096f517782f332a3a7b4cc908e/apps/i18n/applab/en_us.json#L533) an example of a string that uses a parameter. You can see `table` is provided [in `commands.js`](https://github.com/code-dot-org/code-dot-org/blob/d185420ee51c89096f517782f332a3a7b4cc908e/apps/src/applab/commands.js#L1879).

Reply:
ty! fixed in the new PR